SystemServices/SystemServices.cpp

```diff
@@ -3899,13 +3898,15 @@ namespace WPEFramework {
 
                 case IARM_BUS_SYSMGR_SYSSTATE_TIME_SOURCE:
                     {
-                        if (sysEventData->data.systemStates.state)
-                        {
-                            LOGWARN("Clock is set.");
-                            if (SystemServices::_instance) {
-                                SystemServices::_instance->onClockSet();
-                            } else {
-                                LOGERR("SystemServices::_instance is NULL.\n");
+                        if(sysEventData != nullptr){
```

Comment:
please remove changes in SystemServices. asking this 3rd time

you should not make changes in SystemServices. this change makes no sense as well because sysEventData is used in preceding lines